test/integration/sanMvrvLongShortDiffTest.js

```diff
@@ -0,0 +1,50 @@
+const chai = require('chai')
+const expect = chai.expect
+
+const { testFieldTypes } = require('../helper.js')
+
+const {
+  testHandlesNullData,
+  assertNumberOfRecords
+} = require('../integration_helper.js')
+
+const {
+  slug,
+  from,
+  to,
+  days,
+  numberOfDays,
+  formatDate
+} = require('../setup.js')
+
+describe('mvrv_long_short_diff metric', async () => {
+  const expected = {
+    date: 'string',
+    value: 'number'
+  }
+
+  const response = san.SAN_MVRV_LONG_SHORT_DIFF(slug, from, to)
```

Comment:
There is currency argument in the function, what you are not passing it in the test?